packages/story-editor/src/components/audioPlayer/index.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  useState,
+  useEffect,
+  useCallback,
+  useRef,
+} from '@web-stories-wp/react';
+import {
+  THEME_CONSTANTS,
+  themeHelpers,
+  Icons,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+} from '@web-stories-wp/design-system';
+import { __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../tooltip';
+
+const StyledButton = styled(Button)`
+  ${({ theme }) =>
+    themeHelpers.focusableOutlineCSS(
+      theme.colors.border.focus,
+      theme.colors.bg.secondary
+    )};
+`;
+
+const Wrapper = styled.div(
+  ({ theme }) => css`
+    position: relative;
+    min-width: 40px;
+    width: 100%;
+    height: 36px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: ${theme.colors.fg.primary};
+
+    ${themeHelpers.expandPresetStyles({
+      preset: {
+        ...theme.typography.presets.paragraph[
+          THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL
+        ],
+      },
+      theme,
+    })};
+  `
+);
+
+function AudioPlayer({ title, src }) {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const playerRef = useRef();
+
+  const startPlayer = useCallback(() => setIsPlaying(true), [setIsPlaying]);
+  const stopPlayer = useCallback(() => setIsPlaying(false), [setIsPlaying]);
+
+  useEffect(() => {
+    const player = playerRef.current;
+
+    if (!player) {
+      return;
+    }
+
+    if (isPlaying) {
+      player.play();
```

Comment:
Do we need a then catch here?
https://github.com/google/web-stories-wp/blob/833cc68bd99067cc2288f7026a32f2f0060ff9b8/packages/story-editor/src/elements/video/controls.js#L227-L230

Just in case the audio fails to play for some reason.

Reply:
Yeah can't hurt 👍